### ISSUE TITLE
Localization issue

### ISSUE THREAD

**Issue by icBaiye** [nicolaskruchten/pivottable#875]
Greetings.
I find there is something wrong with the localization script, when I add it to my demo, the renderers remain English while others are okay, after reading the source file, I think this code's causing this:

opts = $.extend(true, {}, localeDefaults, $.extend({}, defaults, inputOpts))

Because we wanna enable the plugins, so these codes are used according to the samples:

var renderers = $.extend(
                $.pivotUtilities.renderers,
                $.pivotUtilities.c3_renderers,
                $.pivotUtilities.d3_renderers,
                $.pivotUtilities.export_renderers
            );
            $.getJSON("/static/mps.json", function (mps) {
                $("#div").pivotUI(
                    mps,
                    {
                        renderers: renderers
                    },
                    false,
                    "fr"
                );
            });

This variable renderers is that "inputOpts" above which is not localized, but "defaults" is localized.

I find a way to solve this issue, you DO NOT need to pass the renderers param at all, c3/d3 renders are added automatically, ALSO "Horizontal Bar Chart" and "Horizontal Stacked Bar Chart" are missing in those localization scripts.

**Comment by nicolaskruchten** [comment]
Yes, if you override the `renderers` you get the ones you're overriding with, rather than the ones from the locale file. The `fr` locale file is indeed missing some renderers, as are most locale files. My general recommendation is that you tweak the locale file to your liking :)
